raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -281,35 +281,36 @@ contract TokenNetwork is Utils {
         isOpen(channel_identifier)
         public
     {
-        require(channel_identifier == getChannelIdentifier(participant, partner));
+        uint256 added_deposit;
+        uint256 channel_deposit;
+
+        // Check deposit constraints
         require(total_deposit > 0);
         require(total_deposit <= deposit_limit);
 
-        uint256 added_deposit;
-        uint256 channel_deposit;
+        // Check that address data is correct
```

Comment:
Here we check that:
- participant and partner addresses have a current channel opened
- the channel that they have open is the same as the channel identifier given as input

Otherwise, `address data` does not really explain what we are checking.

Reply:
I described it as general `address data` to suggest to an auditor, that the actual requires are not in that part of the code but inside of `getChannelIdentifier`. I can add that to the comment to make that more explicit.